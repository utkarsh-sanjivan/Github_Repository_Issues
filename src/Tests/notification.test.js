import React from 'react';
import { create } from "react-test-renderer";
import { CloseCircleOutlined } from '@ant-design/icons';
import notify from '../component/Notification/index';

test('Test Notification Component', () => {
  const component = create(notify({
    message: 'Please type in user/repo format',
    placement: 'topRight',
    icon: <CloseCircleOutlined/>,
    }));
  expect(component.toJSON()).toMatchSnapshot()
});