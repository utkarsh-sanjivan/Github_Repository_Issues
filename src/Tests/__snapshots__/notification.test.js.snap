// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Notification Component 1`] = `null`;
